### DISCUSSION TITLE
Jump to a coordinate

### DISCUSSION THREAD

**Discussion by shajahan81** [Helium314/SCEE#548] [Ideas]
When we already know a coordinate, we are unable to jump to a particular location and add a node/POI. Or even complete streets and POI in that area. Please provide an option to jump to.
I resort to Organic maps for this purpose now.

**Comment by mcliquid** [comment]
The following works as a workaround:
Open osm.org and search for the location there. Then click on the Share button in the menu on the right and click on the Geo-URI. In the selection of apps you can select SCEE and the app will start directly at the location.

**Reply by Helium314** [reply]
The geo URI stuff was also my first idea. Unfortunately you can't share geo URIs from Organic Maps, and also in OSMAnd it's somewhat hidden as far as I can remember.

**Reply by mnalis** [reply]
- In OsmAnd, you long press on the map (or choose location in any other way), press share button, and choose `geo:`. 
So it's as visible as anything else in OsmAnd (which is admittedly somewhat complex in general, but this one is even simpler than usual, and beats Organic Maps)

- Organic Maps is more problematic there, as does not have useful sharing by itself AFAIK -- you can only copy coordinates manually from it (by long pressing on the map, pressing on the name, long pressing on crosshairs button and choosing _"Copy xxx.xx, yyy.yy"_ from menu) and then pasting that in destination app search manually.
  But with almost the same effort you could instead choose _"Copy osm.org/go/xxxxx"_, and opening that one in browser will allow workaround suggested by @mcliquid (one could additionally use [Untrackme](https://f-droid.org/en/packages/app.fedilab.nitterizeme/)  to simply that step, as it can be configured to transform maps.google.com and osm.org links to `geo:` URIs automatically)

**Reply by shajahan81** [reply]
Thank you guys for workaround suggestions. Hoping to see the Jump to feature in SCEE and Street Complete.

In Organic maps I was directly adding a place, instead of sharing it to SCEE. Then Edit from SCEE later.
